docs/get-started/getting-started-gcp.md

```diff
@@ -142,7 +142,7 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link. Download the sample 
-[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) and upload it in Jupyter.
+[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL.ipynb) and upload it in Jupyter.
 
 To get the input data of the sample notebook, we need to download the full 17 years' [Mortgage 
```

Comment:
The getting started guides for csp still point to the datasets hosted on rapids.ai 
We can use these [instructions](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.08/docs/get-started/xgboost-examples/dataset/mortgage.md) to download the dataset.

Reply:
updated